microceph/ceph/service_placement_nfs.go

```diff
@@ -0,0 +1,43 @@
+package ceph
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/canonical/microceph/microceph/interfaces"
+)
+
+type NFSServicePlacement struct {
```

Comment:
Service IP is missing that should be used as a `BindAddr` value.

Reply:
Done